<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistogramWindow</class>
 <widget class="QWidget" name="HistogramWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>RGB Histograms</string>
  </property>
  <widget class="QLabel" name="redH">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>256</width>
     <height>90</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>red</string>
   </property>
  </widget>
  <widget class="QLabel" name="greenH">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>105</y>
     <width>256</width>
     <height>90</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>green</string>
   </property>
  </widget>
  <widget class="QLabel" name="bluH">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>256</width>
     <height>90</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>blu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
    </font>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
    </font>
   </property>
   <property name="text">
    <string>G</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>235</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
    </font>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
